openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,10 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class is_usda_disabled(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Indicates whether person is disabled per USDA definition"
```

Comment:
add formula and label

Reply:
what's the difference between documentation and a label? I see that most of the tax variables only have a label, but it seems it fills the role of documentation

Reply:
label is 3-4 words, description is longer.

tax variables don't have descriptions because they've largely been auto-generated from tax-calculator code. This might be a better example (except the wrong currency!):

https://github.com/PolicyEngine/openfisca-us/blob/578e3eb37bf8f1c23d69e80777a6ebd8c162934f/openfisca_us/variables/usda/school_meals.py#L54-L60

Reply:
Yep - @nmodelo you're right about the general state of the tax variables; since we want to exactly preserve tax-calculator logic, we decided the most efficient strategy is to do the translation/porting first, check integrations all work as expected, then go back and restructure/add metadata later.